lib/util/http/index.ts

```diff
@@ -148,8 +150,7 @@ export class Http<Opts extends HttpOptions = HttpOptions> {
     );
 
     const etagCache =
-      httpOptions.etagCache &&
-      (options.method === 'get' || options.method === 'head')
+      httpOptions.etagCache && options.method === 'get'
```

Comment:
why remove head request?

Reply:
It isn't implemented at the type level

Reply:
but it's used by docker datasource, which is doing head requests which needs to be cached

Reply:
I believe this API isn't being used anywhere at the moment

Reply:
Please check if docker is doing head requests directly or via this method

Reply:
It's impossible at the type level to use this `etagCache` option with `head` request, and I've double checked it for docker